src/components/NetworkModal/index.tsx

```diff
@@ -192,6 +192,13 @@ export default function NetworkModal(): JSX.Element | null {
 
     const isMetaMask = window.ethereum && window.ethereum.isMetaMask
```

Comment:
I see.
Though that would make Coinbase not render at all anymore?
or is that good for now?
(because, if only coinbase exists, actually all functions work: network switching, etc)

Reply:
no sorry, this comment of me is not correct